test/Symmetric/OpensslAeadTest.php

```diff
@@ -190,11 +190,11 @@ public function testCcmEncryptWithTagSize()
         $this->crypt->setMode('ccm');
         $this->crypt->setKey(random_bytes($this->crypt->getKeySize()));
         $this->crypt->setSalt(random_bytes($this->crypt->getSaltSize()));
-        $this->crypt->setTagSize(24);
+        $this->crypt->setTagSize(14);
```

Comment:
This test has also been broken for awhile now, but this change fixes it.

Reply:
i saw that before and wondered how it ever worked, the tag size is max 16 for that 
"In CCM, the authentication tag τ is of variable length: it is permitted to be 4, 6, 8, 10, 12, 14, or 16 bytes long."
https://csrc.nist.gov/CSRC/media/Projects/Block-Cipher-Techniques/documents/BCM/Comments/800-38-series-drafts/CCM/RW_CCM_comments.pdf